templates/components/search/quick-results-aria-message.html

```diff
@@ -0,0 +1,6 @@
+<p role="status"
+   aria-live="polite"
+   class="aria-description--hidden"
+   data-search-aria-message
+   data-search-aria-message-predefined-text="{{{lang 'search.results.quick_count_live'}}}"
+></p>
```

Comment:
If we move this partial to quick-search we don't need to create separate file for it

Reply:
When input in focus, browser determines its wrapper as group, and reading all its content including search notification text. So I created this file to import it in two separate navigations